torch/csrc/jit/frontend/schema_matching.cpp

```diff
@@ -556,11 +665,23 @@ std::pair<size_t, MatchedSchema> matchSchemas(
         schemas, loc, graph, args, kwargs, self, /*render_errors=*/true);
   }
 
+  if (matches.size() > 1) {
+    throw ErrorReport(loc) << "The type of values annotated with `Any`"
+                           << " will only be automatically converted "
+                           << "to a more specific type if it's possible"
+                           << " to uniquely identify that type from the"
+                           << " function call. Either change the type "
+                           << "annotation or add an assert statement "
+                           << "before the first use of this value\n"
+                           << "The following variants are available:\n"
+                           << prefixLine(failure_messages.str(), "  ")
+                           << "\nThe original call is";
+  }
```

Comment:
Do you think this will break any tests which may be looking for a specific error message?

Reply:
Good point, I'll make sure to change any tests to match this new error message. I don't feel like we should worry about BC with error messages, though

Reply:
Right, the text in error messages isn't part of BC promise. Updating all tests that are affected by this error message change is good enough.